src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackReviews/FeedbackReviews.hooks.ts

```diff
@@ -0,0 +1,44 @@
+import { useDecryptedActivityData } from 'modules/Dashboard/hooks';
+
+import { AssessmentActivityItem } from '../../RespondentDataReview.types';
+import { GetFeedbackReviewsProps, ReviewData } from './FeedbackReviews.types';
+
+export const useFeedbackReviewsData = () => {
```

Comment:
This hook was moved to `...RespondentDataReview/hooks/useFeedbackReviewsData`, so this is a duplication of the code in the other location.